src/openzeppelin/tests/security/test_pausable.cairo

```diff
@@ -1,7 +1,21 @@
+use option::OptionTrait;
+use starknet::contract_address_const;
+use starknet::ContractAddress;
+use starknet::testing;
+
 use openzeppelin::security::pausable::Pausable;
-use openzeppelin::security::pausable::Pausable::PausableImpl;
 use openzeppelin::security::pausable::Pausable::InternalImpl;
+use openzeppelin::security::pausable::Pausable::PausableImpl;
+use openzeppelin::security::pausable::Pausable::Paused;
+use openzeppelin::security::pausable::Pausable::Unpaused;
 
+
+fn CALLER() -> ContractAddress {
+    contract_address_const::<15>()
+}
+fn ZERO() -> ContractAddress {
+    contract_address_const::<0>()
+}
```

Comment:
do we want to centralize these? it's the same snippets in a lot of files.

Reply:
I agree is better that way.